testbench/test-exec/packages/test-package/test-version/get-cache.py

```diff
@@ -1,5 +1,5 @@
 import json
 import os
 
-cache_value = json.loads(os.getenv("VELOCITAS_CACHE_DATA"))["my_cache_key"]
-print(f"Cache value: {cache_value}")
+cache_value = json.loads(os.getenv("VELOCITAS_CACHE_DATA"))
```

Comment:
Why are we not checking for a certain key anymore? Should be reverted?

Reply:
it's basically like that to check
```javascript
expect(result.stdout).to.not.contain('var');
expect(result.stdout).to.not.contain('asdc');
```
since we have
```python
print("var=\"asdc' >> VELOCITAS_CACHE")
```
inside the set-cache script (taken over from the unit tests)

Reply:
Then we should not call the variable `cache_value` but `cache_content`